modules/ycc_vm.py

```diff
@@ -680,6 +680,7 @@ def _get_instance_params(self, spec):  # pylint: disable=R0914
         metadata = spec.get("metadata")
         labels = spec.get("labels")
         security_groups = spec.get("security_groups")
+        disk_name = hostname + "-disk"  # CST-965: added disk_name to boot disk for Grafana dashboards
```

Comment:
```suggestion
        disk_name = hostname + "-boot"  # CST-965: added disk_name to boot disk for Grafana dashboards
```

Reply:
IMHO it is more informative.

Reply:
Thanks, you are totally wright.
1. Customer asks for -disk pattern in ticket.
2. Do not grafana`s dashboard split disk name on screenshot.
I will ask customer for -boot pattern and change concatenation to f-string

Reply:
Fixed. Thanks.

Reply:
Additionally, may be is would be good to save previous behavior for backward compatibility.
Instead of hard-coding this thing, introduce a new parameter.

Maybe other users would like another behavior, the old one, for example.

Reply:
@Maunty do you mean to add `disk_name` option to `ycc_vm` module (the same we have for `disk_type` and `disk_size`) and use generated one in case of no option defined (assume it is assigned automatically)? It make sense.
_Potentially we can allow to pass `name` into `secondary_disks_spec` and again use generated one in case of no option defined._

In that case caller will set names explicitly. But in that case the name assigning become caller responsibility. We will be forced to modify testing framework logic. Is it ok for us?